.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "max-len": ["error", {"code": 200, "ignoreComments": true}],
+    "no-unused-vars": "off",
+    "camelcase": "off",
```

Comment:
If this is the case for model fields across the board I'm going to have to change a lot of things in my current work. Should be too much effort--tracking them down though will be tough.

Reply:
I think, but am not sure, that this is because the mongo identifiers tend to be `snake_case` rather than `camelCase`. Otherwise `camelCase` is definitely the JS convention, I believe

Reply:
Yes, the models use `snake_case` for attributes (a trait from mongo and the back end). Since those need to be consistent, I disabled this eslint check, so it wouldn't complain about our models.

Reply:
Cool. Went back and I only made a few easily trackable changes to be reverted (which I did), so we're good.